ProjectName/src/screens/Home.tsx

```diff
@@ -1,19 +1,19 @@
 import React, {useState, useEffect} from 'react';
-import {View, StyleSheet, SafeAreaView, Modal, Appearance} from 'react-native';
+import {View, SafeAreaView, Modal, Appearance, AppState} from 'react-native';
 import QuickPay from '../features/quickPay/QuickPay';
 import StarAccount from '../features/accountCard/StarAccount';
 import TransactionList from '../features/transactionList/TransactionList';
 import ServiceMessages from '../features/serviceMessages/ServiceMessages';
-import {isDarkMode} from '../store/themeToggleSlice';
+import {isDarkMode} from '../store/themeToggleSlice/themeToggleSlice';
 import DarkModeStyle from '../theme/DarkModeStyle';
-import {AppState} from 'react-native';
 import BlurScreen from '../components/BlurScreen/BlurScreen';
 import {useAppSelector, useAppDispatch} from '../store/hooks';
-import {Avatars} from '../constants/Avatars.constant';
-
+import {Users} from '../constants/Users.constant';
+import {useThemeToggle} from '../store/themeToggleSlice';
+import {styles} from './Home.style';
 const Home: React.FC = () => {
   const dispatch = useAppDispatch();
-  const darkmode = useAppSelector(state => state.themeToggle);
+  const darkmode = useThemeToggle().isDarkModeState;
```

Comment:
When you use boolean variables for conditions, please use `is, should, will` prefixes. It helps to understand code and intention better.